content/tutorials/python-whirlwind-tour.md

```diff
@@ -335,205 +202,135 @@ Half way there!
 	prices[index] = value * 1.20
 
-print prices
-~~~~~
+print(prices)
+```
 
 Output:
 
-~~~~~ not-code
+```python
 [4.8, 6.0, 2.4, 1.7999999999999998]
-~~~~~
-
-Exercises: Lists and Loops
---------------------------
-
-### A better average calculator ###
-
-Write a program which calculates the average of a list of numbers. You can specify the list in the code.
-
-**Extension:**
-You can tell when a user has not entered anything at a `raw_input` prompt when it returns the empty string, `""`. Otherwise, it returns a string (like "42.5"), which you can turn into a number with the `float` function. For example:
-
-~~~~~ python
-var = raw_input("Enter a number: ")
-if var == "":
-	print "You didn't enter anything!"
-else:
-	print "You entered",float(var)
-~~~~~
-
-Now, extend your program to let the user enter the list of values. Stop asking for new list entries when they do not enter anything at the `raw_input` prompt.
-
-### Fizz buzz ###
-
-Write a program which prints a list of numbers from 0 to 100, but replace numbers divisible by 3 with "Fizz", numbers divisible by 5 with "Buzz", and numbers divisible by both with "Fizz Buzz".
-
-**Hint:** you might find the `range` function from the next section useful.
-
-**Extension:** create a list of numbers, and replace a number with "Fuzz" if it is a multiple of any number in the list.
-
-### Trees and Triangles ###
-
-You can combine (or 'concatenate') strings in Python with the `+` operator:
-
-~~~~~ python
-str = "Hello "
-str = str + "World!"
-print str
-~~~~~
-
-Write a program that asks the user for a number, and then prints a triangle of that height, with its right angle at the bottom left. For example, given the number 3, the program should output:
-
-~~~~~ not-code
-*
-**
-***
-~~~~~
-
-Try the same, but with the right angle in the top-right, like so (again, for input 3):
-
-~~~~~ not-code
-***
- **
-  *
-~~~~~
-
-**Extension:** print out a tree shape of the given size. For example, a tree of size 4 would look like this:
-
-~~~~~ not-code
-   *
-  ***
- *****
-*******
-   *
-   *
-~~~~~
-
-[Calling functions](#calling-functions) {#calling-functions}
--------------------
+```
 
+## Calling Functions
 Functions are pre-written bits of code that can be run ('called') at any point. The simplest functions take no parameters and return nothing. For example, the `exit` function ends your program prematurely:
 
-~~~~~ python
+```python
 x = 10
 while x > 0:
-	print x
+	print(x)
 	x = x - 1
 	if x == 5:
-		exit()  # not supported in repl.it
-~~~~~
+		exit()  # not supported in repl.it!
+```
 
 This will output the numbers 10 to 6, and then stop. Not very useful. However, most functions take input values ('parameters') and output something useful (a 'return value'). For example, the `len` function returns the length of the given list:
 
-~~~~~ python
+```python
 my_list = [42, "BOOMERANG!!!", [0, 3]]
-print len(my_list)
-~~~~~
+print(len(my_list))
+```
 
 Output:
 
-~~~~~ not-code
+```python
```

Comment:
Should this be "text" (or whatever keyword implies no syntax highlighting) instead of "python"?

Whilst the output of a program may be syntactically valid Python, it shouldn't be considered a Python program. IMO presenting source code with highlighting and program output without it gives a clear visual distinction between the two types of verbatim block used in this document.

There's a few other instances of this in the page.

Reply:
hm, actually yes that does sound like a wise idea, will change